pkgs/applications/networking/3proxy/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, coreutils }:
+
+stdenv.mkDerivation rec {
+  pname = "3proxy";
+  version = "0.8.13";
+
+  src = fetchFromGitHub {
+    owner = "z3APA3A";
+    repo = "3proxy";
+    rev = version;
+    sha256 = "1k5rqldiyakhwhplazlhswkgy3psdkpxhn85605ncwaqx49qy8vk";
+  };
+
+  makeFlags = [
+    "INSTALL=${coreutils}/bin/install"
+    "DESTDIR=$(out)"
+  ];
+
+  preConfigure = ''
+    ln -s Makefile.Linux Makefile
+    substituteInPlace Makefile \
+      --replace /usr/local /usr \
+      --replace '$(DESTDIR)$(prefix)' '$(DESTDIR)'
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Tiny free proxy server";
+    homepage = https://github.com/z3APA3A/3proxy;
+    license = licenses.gpl2;
+
```

Comment:
Remove empty line please.

Reply:
Any reason to bother meat people with fixes like this? There is `nixfmt` on the way which will reformat everything nicely.

Reply:
Did anyway.

Reply:
@volth not really, just think if a rebase is in order, it won't hurt to the have the derivation in a neat formating. I don't think just because ```nixfmt``` is being developed, we can just write ugly formatted code, it kinda is also an expression or a project maturity, but that's just my humble opinion.